backend/test/rules/winner_test.clj

```diff
@@ -29,6 +29,36 @@
       (fn [i] 0)
       (fn [i] 0))))
 
+(defexpect row-winner
```

Comment:
This file is getting long 🙃

Reply:
Yes, we still need to watch it, and it might be useful to split on certain conditions (like draws, player1-wins, player2-wins or whatever)